java/libphonenumber/pom.xml

```diff
@@ -30,6 +30,26 @@
     </testResources>
     <plugins>
       <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <version>3.2.0</version>
+        <configuration>
+          <instructions>
+            <Export-Package>com.google.i18n.phonenumbers</Export-Package>
+            <Private-Package>com.google.i18n.phonenumbers.data</Private-Package>
```

Comment:
We only have resource files in the data folder; same above - is this intended to cover them? but they don't have a "package" per se?